tldextract/cli.py

```diff
@@ -49,6 +51,14 @@ def main():
     elif len(args.input) is 0:
         parser.print_usage()
         exit(1)
+    elif args.domain:
+        for i in args.input:
+            ext = tld_extract(i)
+            if ext.suffix:
```

Comment:
When `suffix` not present in PSL it contains empty string when `domain` contains actual notavalidsuffix, and `subdomain` contains sub.domains.domain.